### PR TITLE
Suppress sessions

### PR DESCRIPTION
**Pull request by Cymaphore**
Ability to supress session handling in dav requests, see https://github.com/owncloud/core/issues/5383 ... I would recommend to add this to the 5.x-fork as well.

### CODE REVIEW

File: caldav.php [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <?php
+define("DISABLE_SESSION", 1);
```
Comment (Niduroki) [inline caldav.php:2]: What if we don't want to disable sessions?

I'd say this should be a config.php setting, for those who need it.

**Comment by jancborchardt** [comment]
@DeepDiver1975 @ringmaster your calls on this session handling solution please.

**Comment by DeepDiver1975** [comment]
@jancborchardt we need to discus this in the core issue: https://github.com/owncloud/core/issues/5383

**Comment by DeepDiver1975** [comment]
> @jancborchardt we need to discus this in the core issue: owncloud/core#5383

we need to walk a different path

**Pull request closed without merging**